R/SpatialExperiment.R

```diff
@@ -271,13 +271,15 @@ SpatialExperiment <- function(...,
     msg <- function(.) message(sprintf(paste(                
         "both '%s' and '%sNames'  have been supplied; using '%s'.",
         "Set either to NULL to suppress this message"), ., ., .))
+        ## ^ this message 
+        ## is printed also in case the spatialCoords and CoordsNAmes are not 
```

Comment:
Minor typo here: "CoordsNames" instead of "CoordsNAmes"

Reply:
done